lldb/source/API/SBAddressRange.cpp

```diff
@@ -64,7 +64,7 @@ bool SBAddressRange::operator!=(const SBAddressRange &rhs) {
 void SBAddressRange::Clear() {
   LLDB_INSTRUMENT_VA(this);
 
-  m_opaque_up.reset();
+  m_opaque_up->Clear();
```

Comment:
This is no longer needed right? This PR contains this fix right? https://github.com/llvm/llvm-project/pull/95997

Reply:
correct